octodns/provider/ovh.py

```diff
@@ -32,8 +34,8 @@ class OvhProvider(BaseProvider):
 
     SUPPORTS_GEO = False
 
-    SUPPORTS = set(('A', 'AAAA', 'CNAME', 'MX', 'NAPTR', 'NS', 'PTR', 'SPF',
-                    'SRV', 'SSHFP', 'TXT'))
+    SUPPORTS = set(('A', 'AAAA', 'CNAME', 'DKIM', 'MX', 'NAPTR', 'NS', 'PTR',
```

Comment:
I think the `DKIM` isn't needed here.

Reply:
I have put DKIM here, because I reuse SUPPORTS in populate:

```
if _type not in self.SUPPORTS:
    ****
    continue
```

Reply:
Ah. Didn't catch that. The main use-case for `SUPPORTS` is [in the base source/provider](https://github.com/github/octodns/blob/1a0055eedb6acc4c8b2319efd7208544d47bdb11/octodns/source/base.py#L38-L39). The secondary/related use is for octoDNS users to see what record types the provider can work with. That kind of puts this in a grey area since `DKIM` wouldn't apply to the first, but would to the second. 

I guess it makes sense to leave it in, but if you don't mind please add a comment above the `SUPPORTS` variable saying why it's there b/c it's a type specific to OVH that the provider supports by converting to `TXT`.

Reply:
I have added a comment explaining that SUPPORTS is used in populate method